commands/releases/info.js

```diff
@@ -14,31 +15,56 @@ function* run (context, heroku) {
     let releases = yield heroku.request({
       path: `/apps/${context.app}/releases`,
       partial: true,
-      headers: {Range: `version ..; max=2, order=desc`},
+      headers: {
+        Range: `version ..; max=2, order=desc`,
```

Comment:
I might've wrote this, but why are we fetching 2 here? Couldn't we just fetch 1? (though the performance should be the same)